test/features/user_records_an_application_test.rb

```diff
@@ -1,7 +1,22 @@
 require './test/test_helper'
 
 class UserRecordsAnApplication < ActionDispatch::IntegrationTest
+
+  def with_js_driver
+    Capybara.current_driver = :selenium
+    yield
+    Capybara.use_default_driver
+  end
+
+  def test_populates_new_application_form_from_query_param
+    with_js_driver do
+      visit new_application_path(uri: "google.com/jobs/1")
+      assert_equal "google.com/jobs/1", find_field('URL of the Job Posting*').value
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.